tests/Ticket/DC521TestCase.php

```diff
@@ -101,42 +99,43 @@ public function testIdOnlyPgsqlAutoIncrementField()
         $m->save();
         $this->assertEqual($m->id, 1);
     }
-    
+
     public function testIdOnlyPgsqlAutoIncrementFieldWithDefinedValue()
     {
-        $m = new DC521IdOnlyTestModel();
+        $m     = new DC521IdOnlyTestModel();
         $m->id = 9999;
         $m->save();
         $this->assertEqual($m->id, 9999);
     }
-    
+
     public function testPgsqlAutoIncrementFieldWithDefinedValue()
     {
-        $m = new DC521TestModel();
+        $m     = new DC521TestModel();
         $m->id = 9999;
         $m->save();
         $this->assertEqual($m->id, 9999);
     }
-    
+
     public function testPgsqlAutoIncrementFieldWithDefinedValueAndData()
     {
         $m = new DC521TestModel();
         $this->assertEqual($m->id, null);
-        $m->id = 111111;
-        $m->data = "testdata";
+        $m->id   = 111111;
+        $m->data = 'testdata';
         $m->save();
         $this->assertEqual($m->id, 111111);
     }
-    
-    public function prepareTables() 
+
+    public function prepareTables()
     {
         $this->tables = array('DC521TestModel', 'DC521IdOnlyTestModel');
         parent::prepareTables();
     }
-    
+
     public function tearDown()
     {
         Doctrine_Manager::resetInstance();
-    	   $this->driverName = null;
-    	   parent::tearDown();
+        $this->driverName = null;
+        parent::tearDown();
     }
+}
```

Comment:
php-cs-fixer didn't initially process this file because it was missing the closing `}` here, so I added that to fix the syntax error. I **assume** this test wasn't being included in the suite at all (probably would've caused the tests to error if it had been). I haven't tried adding it.

Reply:
This test is explicitly excluded in `run.php` as it's postgres specific.